Userland/Libraries/LibPDF/Fonts/CFF.cpp

```diff
@@ -138,105 +138,247 @@ PDFErrorOr<NonnullRefPtr<CFF>> CFF::create(ReadonlyBytes const& cff_bytes, RefPt
     return cff;
 }
 
-HashMap<CFF::SID, DeprecatedFlyString> CFF::builtin_names {
-    { 0, ".notdef" },
-    { 1, "space" },
-    { 9, "parenleft" },
-    { 10, "parenright" },
-    { 13, "comma" },
-    { 14, "hyphen" },
-    { 15, "period" },
-
-    { 17, "zero" },
-    { 18, "one" },
-    { 19, "two" },
-    { 20, "three" },
-    { 21, "four" },
-    { 22, "five" },
-    { 23, "six" },
-    { 24, "seven" },
-    { 25, "eight" },
-    { 26, "nine" },
-    { 27, "colon" },
-    { 28, "semicolon" },
-
-    { 34, "A" },
-    { 35, "B" },
-    { 36, "C" },
-    { 37, "D" },
-    { 38, "E" },
-    { 39, "F" },
-    { 40, "G" },
-    { 41, "H" },
-    { 42, "I" },
-    { 43, "J" },
-    { 44, "K" },
-    { 45, "L" },
-    { 46, "M" },
-    { 47, "N" },
-    { 48, "O" },
-    { 49, "P" },
-    { 50, "Q" },
-    { 51, "R" },
-    { 52, "S" },
-    { 53, "T" },
-    { 54, "U" },
-    { 55, "V" },
-    { 56, "W" },
-    { 57, "X" },
-    { 58, "Y" },
-    { 59, "Z" },
-    { 66, "a" },
-    { 67, "b" },
-    { 68, "c" },
-    { 69, "d" },
-    { 70, "e" },
-    { 71, "f" },
-    { 72, "g" },
-    { 73, "h" },
-    { 74, "i" },
-    { 75, "j" },
-    { 76, "k" },
-    { 77, "l" },
-    { 78, "m" },
-    { 79, "n" },
-    { 80, "o" },
-    { 81, "p" },
-    { 82, "q" },
-    { 83, "r" },
-    { 84, "s" },
-    { 85, "t" },
-    { 86, "u" },
-    { 87, "v" },
-    { 88, "w" },
-    { 89, "x" },
-    { 90, "y" },
-    { 91, "z" },
-
-    { 104, "quotesingle" },
-    { 105, "quotedblleft" },
-
-    { 111, "endash" },
-
-    { 116, "bullet" },
-
-    { 119, "quotedblright" },
-
-    { 137, "emdash" },
-
-    { 170, "copyright" },
+/// Appendix C: Predefined Charsets
+Vector<DeprecatedFlyString> CFF::builtin_names {
```

Comment:
If this never changes during runtime, an AK:: Array might be more appropriate.

Reply:
Ah, thanks, that's true. I also vaguely know there's a fixed vector class or something similar, buy in full honesty I didn't bother to check. I'll change it to an Array later when I get some time, thanks!

Reply:
These static ones can probably also just be stored as StringViews here

Reply:
Yes they could -- but then they'll end up being interned anyway later on, or at least those that end up being used. If it's worth then I can change it.

Reply:
It's a little nicer because with StringViews you don't need to stay the size of the array, i.e. you can just do this:
```
Array CFF::builtin_names {
    ".notdef"sv,
    "space"sv,
    "exclam"sv,
    "quotedbl"sv,
```
...

Reply:
OK, I've changed it to be an Array of StringViews in the end. To simplify things I had to remove the declaration of the static variable from the `CFF` class, and define it at the global scope instead (well, still within the PDF namespace), otherwise the declaration in the CFF class (which still had to be explicitly `Array<StringView, 229>`) didn't match the type-deduced one (`Array<...auto...>`). Probably a good simplification anyway.